src/styles.scss

```diff
@@ -0,0 +1,132 @@
+/* You can add global styles to this file, and also import other style files */
+
+@import "bootstrap.min";
+// @import '~bootstrap/dist/css/bootstrap.min.css';
+@import url('https://fonts.googleapis.com/css?family=Roboto:300,400,700');
```

Comment:
or you can add it in the index.html `<link href="https://fonts.googleapis.com/css?family=Roboto:300,400,700" rel="stylesheet" />`

Reply:
I will explain